SinergiaTIC/SinergiaDA
# Maintenance merge

* dashboard - custom css

* xxx

xxx

* mysql - return poolConnection

* fix port number

* sda_hidden column

visualizar o no columnas con factor sda_hidden para usuarios Admin

* test

* remove filter issue fix

* poolLimit

en base_datamodel & datasrorce.controller & updateModel,controller

* colmun hidden fix

* SDA-57 Cambio del ? por el lápiz

* dashboard - queryParams noWheels

* mysql connection - pool

* afegit nowheel=true per ocultar la rodeta d'opicons al dashboards

* Ajuste de los filtros

* mysql - pool connection

* Check connection

* SDA-78 : N/A user agregado para evitar usuarios no definidos

* SDA 77: discriminación por modelo añadido

* SDA #78: añadido usuario desconocido para no bloquear lista

* updateModel grantedRoles cleaner

* Revert "Merge branch 'reporting' into master"

This reverts commit 3065777bbb50c461cc6ded961f077ca15cc804cc, reversing changes made to 292fd3b3e86cad28e993f79fe12c09c4852d8c00.

* minor fixes

* restauración

* Revert "restauración"

This reverts commit 5aa6c4150fd748232b2be310af4a0693ca16446e.

* gitignore

* Syncro

* ignore

* Pool Descartado por el momento

* Polski I

* Query Parameteres per anonymous login

* Restore config

* Ignore

* SDA-61 Los campos value list deben ser left join

* Esto no va aqui

* No Data Bug

* Cambios menores

* SDA-80 Limitar el número de tablas en un datasource

* Revert "Merge branch 'reporting' into master"

This reverts commit 8cb8da9e2da6884eb93f5490c96071e698ddfe9c, reversing changes made to bd1a728d86ce3a999d858c4d618fd90adb58007b.

* Revert "Merge branch 'reporting' into master"

This reverts commit 8cb8da9e2da6884eb93f5490c96071e698ddfe9c, reversing changes made to bd1a728d86ce3a999d858c4d618fd90adb58007b.

# Conflicts:
#	eda/eda_api/lib/services/query-builder/query-builder.service.ts #	eda/eda_app/src/app/module/components/eda-panels/eda-blank-panel/eda-blank-panel.component.html

* Revert "Merge branch 'reporting' of https://github.com/SinergiaTIC/SinergiaDA into reporting"

This reverts commit 1ad847f23a346827c90328232370a2dfd0015cef, reversing changes made to d730d60e17dd2302bdd1097e21f7d7ccdda9d963.

* Revert "Revert "Merge branch 'reporting' into master""

This reverts commit d730d60e17dd2302bdd1097e21f7d7ccdda9d963.

* Syncro Test 1

* Revert "Revert "Revert "Merge branch 'reporting' into master"""

This reverts commit 26b1758f05233e0cf30dbc34b48f14004edfa735.

* xx

* SDA- 80

* SDA -  61

* SDA-61 Los campos value list deben ser left join

* A ver si ahora ya si que si

* MySql a 500 otra vez

* SDA-80 Limite de 400 tablas para Mysql

* base datamodel restpored

* SDA-122 Evitar errores en el caso de que una columna desaparezca misteriosamente

* SDA-122 Errro cuando falta un campo de los filtros en el modelo.

* SDA-61 Campos multienum con tabla puente

* Refactorización de vistas de postgres

* limieza

* limpieza

* SDA-77 columnas ocultas

* SDA-77 A la segunda

* polski by nadia

* polski build

* Polski login

* Revisión PL y update para 2.0

* slice en vez de substring

* locales

* polski sidebar

* Seguridad :: Admin lo ve todo

* corrección cosmética

* polish

* Bug permisos

* Final merge

---------
